demisto_sdk/commands/validate/file_validator.py

```diff
@@ -1129,7 +1129,7 @@ def get_content_release_identifier(self):
         except Exception:
             return
         else:
-            return file_content.get('jobs').get('build').get('environment').get('GIT_SHA1')
+            return file_content.get('references').get('environment').get('environment').get('GIT_SHA1')
```

Comment:
```suggestion
            return file_content.get('references', {}).get('environment', {}).get('environment', {}).get('GIT_SHA1')
```

Reply:
Don't you think it'll be better if it actually fails if it fails in this step?

Reply:
on `NoneType has no get`? no..

Reply:
yeah, but it means that it got the circle.yml as it needs to then the key is wrong.

Reply:
more likely is that we will encounter it before it'll reach any customer

Reply:
doesnt matter.. if something is wrong, need to have an indicative error message